tests/integration/modules/beacons.py

```diff
@@ -0,0 +1,63 @@
+# coding: utf-8
+
+# Python Libs
+from __future__ import absolute_import
+
+# Salt Libs
+from salt.modules import beacons
+import integration
+from salttesting import skipIf
+from salttesting.helpers import destructiveTest, ensure_in_syspath
+
+ensure_in_syspath('../../')
+
+beacons.__opts__ = {}
+
+
+class BeaconsTest(integration.ModuleCase):
+    '''
+    Tests the beacons execution module
+    '''
+    @destructiveTest
+    def test_add(self):
+        ret = self.run_function('beacons.add', ['ps', {'apache2': 'stopped'}])
+        self.assertEqual(ret, {'comment': 'Added beacon: ps.', 'result': True})
+
+    @destructiveTest
+    def test_list(self):
+        ret = self.run_function('beacons.list')
+        self.assertEqual({'beacons': {}}, ret)
```

Comment:
Tests do not run in the order that they are written...it might make sense to make sure you're adding/removing beacons in a single test case. I think in this case it was unintended that the `test_add` and `test_delete` tests ran _before_ the `test_list`, resulting in an empty dict.